evm/deploy-its.js

```diff
@@ -349,6 +362,10 @@ async function deployAll(config, wallet, chain, options) {
             contractConfig[key] = contract.address;
         }
 
+        if (isHyperliquidChain(chain) && shouldUseBigBlocks(key)) {
+            await switchHyperliquidBlockSize(options, false, chain);
+        }
```

Comment:
not robust to depend on a non-atomic revert after deployment. E.g. the deployment of the implementation fails in the middle. When you trigger the deployment again, the same account is still set in big block mode. Instead, use try/finally to always revert even if there's an exception on deploy?
```
try {
   const contract = await deployment.deploy();
} finally {
   switch back
}
```